mapcss/mapcss2osmose.py

```diff
@@ -71,6 +72,22 @@ def simple_selector_pseudo_class(t, c):
     t['pseudo_class'] = list(filter(lambda p: not (p['not_class'] and p['pseudo_class'] in ('completely_downloaded', 'in-downloaded-area')), t['pseudo_class']))
     return t
 
+def convert_area_selectors(t, c):
+    """
+    type = rule
+    Convert area* rules to way[area!=no]* + relation[type=multipolygon]*
+    """
+    areaselectors = list(filter(lambda selector: selector['simple_selectors'][0]['type_selector'] == 'area', t['selectors']))
+    for s in areaselectors:
+        relSelector = deepcopy(s)
+        relSelector['simple_selectors'][0]['type_selector'] = 'relation'
+        relSelector['simple_selectors'][0]['predicates'].append({'type': 'booleanExpression', 'operator': '=', 'operands': [{'type': 'valueExpression', 'operator': None, 'operands': [{'type': 'primaryExpression', 'derefered': False, 'value': {'type': 'osmtag', 'value': 'type'}}]}, {'type': 'valueExpression', 'operator': None, 'operands': [{'type': 'primaryExpression', 'derefered': False, 'value': {'type': 'osmtag', 'value': 'multipolygon'}}]}], 'selector_index': selector_index_map['arearule']}) # This is the output of an [type=multipolygon] selector with a mock selector_index
```

Comment:
Hard coding this may make it hard to maintain ?

Reply:
I fully agree, but I could not find a way to soft-code it. Preferably one would just want `somefunction("[type=multipolygon]")` (or even extract it from `way[type=multipolygon] { throwWarning: "";}`)
The parser requires a ANTLR filestream, which is only defined for text files; even using an StringIO failed.

Reply:
(If you can think of a way, also the similar hardcoded rule of pseudo_class_righthandtraffic could be rewritten)

Reply:
Have an idea :)

Reply:
It works :)
(Let's do pseudo_class_righthandtraffic in a different PR)